### ISSUE TITLE
Can't create big archive (more then 4GB)

### ISSUE THREAD

**Issue by maslbl4** [srikanth-lingala/zip4j#246]
OS: Windows 10
Version: 2.6.3

If add files one by one and the zip archive exceeds 4 GB then an exception will be thrown

```
Caused by: net.lingala.zip4j.exception.ZipException: java.io.IOException: Negative seek offset
	at net.lingala.zip4j.tasks.AsyncZipTask.performTaskWithErrorHandling(AsyncZipTask.java:53)
	at net.lingala.zip4j.tasks.AsyncZipTask.execute(AsyncZipTask.java:40)
	at net.lingala.zip4j.ZipFile.addFiles(ZipFile.java:292)
	at net.lingala.zip4j.ZipFile.addFile(ZipFile.java:232)
	... 8 more
Caused by: java.io.IOException: Negative seek offset
	at java.io.RandomAccessFile.seek(RandomAccessFile.java:555)
	at net.lingala.zip4j.io.outputstream.SplitOutputStream.seek(SplitOutputStream.java:184)
	at net.lingala.zip4j.tasks.AbstractAddFileToZipTask.initializeOutputStream(AbstractAddFileToZipTask.java:166)
	at net.lingala.zip4j.tasks.AbstractAddFileToZipTask.addFilesToZip(AbstractAddFileToZipTask.java:67)
	at net.lingala.zip4j.tasks.AddFilesToZipTask.executeTask(AddFilesToZipTask.java:26)
	at net.lingala.zip4j.tasks.AddFilesToZipTask.executeTask(AddFilesToZipTask.java:15)
	at net.lingala.zip4j.tasks.AsyncZipTask.performTaskWithErrorHandling(AsyncZipTask.java:46)
	... 13 more
```

Code for reproduce:
``` java
ZipFile zip= new ZipFile(archivePath);
for (File file : folderDir.toFile().listFiles()) { 
    zip.addFile(file); 
} 
```

The workaround is to use method `addFiles()` 

``` java
ZipFile zip= new ZipFile(archivePath);
zip.addFiles(Arrays.asList(folderDir.toFile().listFiles());
```

**Comment by srikanth-lingala** [comment]
Looking into it.

**Comment by srikanth-lingala** [comment]
Fixed in v2.6.4 released today
